pbm/backup/backup.go

```diff
@@ -589,8 +588,18 @@ func newDump(curi string, conns int) *mdump {
 	}
 }
 
+// "logger" for the mongodup's ProgressManager.
+// need it to be able to write new progress data in a new line
+type progressWriter struct{}
+
+func (*progressWriter) Write(m []byte) (int, error) {
+	log.Printf("%s", m)
+	return len(m), nil
+}
+
 // Write always return 0 as written bytes. Needed to satisfy interface
```

Comment:
This comment is now non-applicable, right?

Reply:
No, the comment is still applicable. It is related to the WriteTo method. And we still don't know how many bytes were written by the mongodump. We only could know how many bytes were written by the progress manager (logger let's say) of mongodump.